lib/ddtrace/ext/net.rb

```diff
@@ -1,6 +1,8 @@
 module Datadog
   module Ext
     module NET
+      ENV_REPORT_HOSTNAME = 'DD_TRACE_REPORT_HOSTNAME'.freeze
+      TAG_HOSTNAME = '_dd.hostname'.freeze
```

Comment:
I am wondering if `Ext::NET` is the best place for this or not. It really related to the network at all, only that we use socket to get the hostname (for now).

Do we need a more generic `Ext::Tracer`... ? or something maybe?

This is not a strong opinion, just a semantic suggestion.

Reply:
I see what you mean. I've been categorizing these based on the kind of data, which is why I thought `Net` appropriate. I'm open to other suggestions but `Ext::Tracer` is too generic (everything in the library is related to the tracer) and will end up becoming a catch-all. I'll consider any other ideas though.

Reply:
We just have a `constants.py` in `dd-trace-py` so was easy for us.

Does this need to be Ext/"external"?

Reply:
Yes, its the standard we've been maintaining for this tracer. These constants are often imported piecemeal by integrations and other modules.